vendor.mod

```diff
@@ -62,7 +62,7 @@ require (
 	github.com/moby/sys/sequential v0.5.0
 	github.com/moby/sys/signal v0.7.0
 	github.com/moby/sys/symlink v0.2.0
-	github.com/moby/term v0.0.0-20221120202655-abb19827d345
+	github.com/moby/term v0.0.0-20221205130635-1aeaba878587
```

Comment:
Can we have a tag?

Reply:
@AkihiroSuda sorry, forgot about this one; I recall I had some changes left to do for that module that I wanted to have in before I tagged; I just split/dusted them off and opened PRs https://github.com/moby/term/pulls

Reply:
Tagged a v0.5.0 release